deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -63,6 +63,15 @@ data:
             hosts:
               - 'kube-state-metrics:8080'
             period: 10s
+          - data_stream:
+              dataset: kubernetes.state_job
+              type: metrics
+            metricsets:
+              - state_job
+            add_metadata: true
+            hosts:
+              - 'kube-state-metrics:8080'
+            period: 10s
```

Comment:
Did I miss adding tests for this? I must admit that I don't fully understand the context it's used in.

Reply:
Nope, it' just fine as is. However could you please move this block in the right place preserving the alphabetical order (after state_deployment and before state_node)?

For reference this is the configuration that elastic-agent will use in standalone mode so as to spawn these datastreams/metricsets accordingly. These datastream blocks will be sent by Agent as configs to Metricbeat via grpc so as Metricbeat to start the corrensppoding metricsets under the hood. Hope it makes sense 😬 .

Reply:
Nope, it' just fine as is. However could you please move this block in the right place preserving the alphabetical order (after state_deployment and before state_node)?

For reference this is the configuration that elastic-agent will use in standalone mode so as to spawn these datastreams/metricsets accordingly. These datastream blocks will be sent by Agent as configs to Metricbeat via grpc so as Metricbeat to start the corrensppoding metricsets under the hood. Hope it makes sense 😬 .

Reply:
It makes a great deal of sense. Thanks for explaining. :D

Reply:
👍🏼